tools/pipelines/templates/build-npm-package.yml

```diff
@@ -456,82 +439,19 @@ extends:
                       testResultDirs: '${{ parameters.testResultDirs }}'
 
               # Pack
-              - ${{ if ne(parameters.taskPack, false) }}:
-                  - task: Bash@3
-                    displayName: npm pack
-                    env:
-                      PACKAGE_MANAGER: '${{ parameters.packageManager }}'
-                      RELEASE_GROUP: '${{ parameters.tagName }}'
-                      STAGING_PATH: $(Build.ArtifactStagingDirectory)
-                    inputs:
-                      targetType: filePath
-                      workingDirectory: '${{ parameters.buildDirectory }}'
-                      filePath: $(Build.SourcesDirectory)/scripts/pack-packages.sh
-
-                  # At this point we want to publish the artifact with npm-packed packages, and the one with test files,
-                  # but as part of 1ES migration that's now part of templateContext.outputs below.
-
-              # Collect/publish/run bundle analysis
```

Comment:
No more bundle analysis in this template so all tasks under this condition just go away.